### ISSUE TITLE
Incorrectly ordered MAC addresses in version 5.01

### ISSUE THREAD

**Issue by jasperem** [sflow/sflowtool#23]
Hello again,

I convert saved flow samples to CSV with `sflowtool -l -r file.pcap` and get following results with version 3.41

```
FLOW,192.168.5.1,12345,12346,abcdef123456,abcdef123457,0x0800,1,1,192.168.2.3,172.168.3.4,17,0x00,254,80,28449,0x10,1522,1500,1024
```

and with version 5.01 

```
FLOW,192.168.5.1,12345,12346,badcfe214365,badcfe214375,0x0800,1,1,192.168.2.3,172.168.3.4,17,0x00,254,80,28449,0x10,1522,1500,1024
```

but abcdef123456 and badcfe214365 are not the same MAC addresses.

Please can you fix this.

**Comment by sflow** [comment]
Thanks.  It was a byte-swapping bug introduced when tuning hex-encoding performance.  Fixed in 5.02.

**Comment by jasperem** [comment]
I have to thank you!
